packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/TabPanels/PaymentTabPanel/ManualPaymentRecordDetails.tsx

```diff
@@ -0,0 +1,123 @@
+import { components } from '@/api';
+import { useCurrencies } from '@/core/hooks';
+import { useEntityUserByAuthToken } from '@/core/queries';
+import { MoniteCard } from '@/ui/Card/Card';
+import { useDateTimeFormat } from '@/utils';
+import { t } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import DriveFileRenameOutlineIcon from '@mui/icons-material/DriveFileRenameOutline';
+import { Alert, Box, Skeleton, Typography } from '@mui/material';
+
+import { PaymentRecordDetails } from './RecordManualPaymentModal';
+
+type Props = {
+  paymentRecords: PaymentRecordDetails;
+  invoice: components['schemas']['InvoiceResponsePayload'];
+};
+
+export const ManualPaymentRecordDetails: React.FC<Props> = ({
+  paymentRecords,
+  invoice,
+}) => {
+  const { i18n } = useLingui();
+  const dateTimeFormat = useDateTimeFormat();
+  const { formatCurrencyToDisplay } = useCurrencies();
+
+  const dateTime = i18n.date(
+    new Date(paymentRecords.payment_date ?? ''),
+    dateTimeFormat
+  );
+  // const dateTimeWithReplacedTime = new Date(
+  //   paymentRecords.payment_date ?? ''
+  // ).setHours(
+  //   paymentRecords.payment_time?.getHours() ?? 0,
+  //   paymentRecords.payment_time?.getMinutes() ?? 0
+  // );
+
+  // const dateTime = i18n.date(
+  //   new Date(dateTimeWithReplacedTime),
+  //   dateTimeFormat
+  // );
```

Comment:
Is this intentional?

Reply:
Yes, left for a future iteration. Following our earlier conversations on planning on a strategy for managing timezones across the app.